scripts/build-xcframework.sh

```diff
@@ -16,22 +15,51 @@ generate_xcframework() {
     local configuration_suffix="${4-}"
     local createxcframework="xcodebuild -create-xcframework "
     local GCC_GENERATE_DEBUGGING_SYMBOLS="YES"
-    
+    local OTHER_LDFLAGS="-Wl,-make_mergeable" # Default value with -make_mergeable flag
```

Comment:
l: Do we still need this flags after enabling `MERGEABLE_LIBRARY` in the project settings?